### PR TITLE
Nightly: Update to Kubernetes 1.10

### PR DESCRIPTION
**Pull request by eloycoto**
- Added kubernetes 1.9 in Nightly Jobs

Fix https://github.com/cilium/cilium/issues/3106
Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/3335)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-nightly

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -49,14 +49,21 @@ pipeline {
...
-                        sh 'cd ${TESTDIR}; K8S_VERSION=1.7 ginkgo --focus=" K8s*" -v -noColor'
-                    },
-                   "K8s-1.8":{
-                        sh 'cd ${TESTDIR}; K8S_VERSION=1.8 ginkgo --focus=" K8s*" -v -noColor'
-                    }
-                )
+                script {
+                    parallel(
+                        "K8s-1.7":{
+                            sh 'cd ${TESTDIR}; K8S_VERSION=1.7 ginkgo --focus=" K8s*" -v -noColor'
```
Comment (ianvernon) [inline Jenkinsfile.nightly:55]: Just wondering, how much load can the Jenkins agents handle w.r.t. concurrently running VMs?

Comment (eloycoto) [reply Jenkinsfile.nightly:55]: More than four servers things are getting slower.

Comment (ianvernon) [reply Jenkinsfile.nightly:55]: Great, thanks!

**Review by ianvernon — APPROVED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-nightly

**Comment by eloycoto** [comment]
Seen the same `cgroups` issue: 
```
 s: "Network policy cannot be imported prefix=d8887f5f: cannot get the revision OCI runtime exec failed: exec failed: container_linux.go:348: starting container process caused \"process_linux.go:90: adding pid 23376 to cgroups caused \\\"failed to write 23376 to cgroup.procs: write /sys/fs/cgroup/cpu,cpuacct/kubepods/besteffort/pode95becea-325d-11e8-8317-0800279ce2d5/67a34a77d9e4d21d668c5d15bb5a4771ab9ba38d4fa6871ac45009db641fa9f3/cgroup.procs: invalid argument\\\"\": unknown\r\ncommand terminated with exit code 126\n",
```

**Comment by eloycoto** [comment]
Rebased with master to check with Kubernetes 1.10.0 instead of beta.

**Comment by eloycoto** [comment]
test-nightly

**Comment by ianvernon** [comment]
@eloycoto since you added the wip label I removed the pending-review label. Please add the pending-review label when it is ready for review again.

**Comment by ianvernon** [comment]
@eloycoto any update on this?

**Comment by eloycoto** [comment]
need to be rebased/update to zip ;-)

**Comment by eloycoto** [comment]
Close this one, I'll fix this one when the new Jenkins jobs are in place.

**Pull request closed without merging**